### PR TITLE
Add user feedback template for Design Research

### PR DESCRIPTION
**Pull request by abbycross**
Working in conjunction with @tobi-lasisi to develop this issue template

### CODE REVIEW

**Review by Eric-Arellano — APPROVED** [review]
Lgtm! Not sure if you want Tobi to sign off before merging.

**Comment by abbycross** [comment]
She reviewed it offline and is good with it!

**Pull request merged**
